subsys/bluetooth/services/ots/ots_l2cap.c

```diff
@@ -105,11 +105,18 @@ static void l2cap_disconnected(struct bt_l2cap_chan *chan)
 	LOG_DBG("Channel %p disconnected", chan);
 }
 
+static int l2cap_recv(struct bt_l2cap_chan *chan, struct net_buf *buf)
+{
+	LOG_WARN("Channel %p received %p len %zu", chan, buf, buf->len);
```

Comment:
I guess this should be `BT_WARN()` (and besides, `LOG_WARN` doesn't even exist, like you can see from the CI failure)

Reply:
OTS has been implemented using `LOG_X` instead of `BT_X`. I suggest that if we want to change to `BT_X` then we do it for the entire service implementation. 

Regarding `LOG_WARN` it should be `LOG_WRN`.

Reply:
This code is now removed, since another PR added the callback with functionality in it.